### ISSUE TITLE
[Effect Suggestion] Bodyguard

### ISSUE THREAD

**Issue by DEATHCON1** [gta-chaos-mod/ChaosModV#1504]
**Name of effect**
Personal Bodyguard
Protect The Target

**Effect Description**
Spawns a random npc that follows you, is defenseless and won't fight back (they basically act like amanda when you rescue her over by Didiersachs). If the npc dies, the player dies. There could also be a blue arrow above the npc's head to help identify them better.

**Comment by TheRealAyCe** [comment]
Escort Mission

**Comment by pogue** [comment]
Don't we already have this with spawn random companion?

**Comment by DEATHCON1** [comment]
No, the npc that comes out of 'Spawn Random Companion' does fight and when they die, you don't die.

**Comment by DEATHCON1** [comment]
> Don't we already have this with spawn random companion?

No, with 'Spawn Random Companion', the npc is able to fight and when they die, you don't die...
